docs/administration-guide/databases/bigquery.md

```diff
@@ -1,25 +1,111 @@
+
+2. **Fill out the service account details**. Name the service account, and add a description (the service account ID will populate once you add a name.) Then click the **Create** button.
+
+3. **Grant service account access to this project**. You'll need to add **roles** to the service account so that Metabase will have permission to view and run queries against your dataset. Make sure you add the following roles to the service account:
+
+    - BigQuery Data Viewer
+    - BigQuery Metadata Viewer
+    - BigQuery Job User (distinct from BigQuery User)
+
+    For more information on **roles** in BigQuery, see [Google Cloud Platorm's documentation](https://cloud.google.com/bigquery/docs/access-control).
+
+4. **Create key**. Once you have assigned roles to the service account, click on the **Create Key** button, and select **JSON** for the **key type**. The JSON file will download to your computer.
+
+> **You can only download the key once**. If you delete the key, you'll need to create another service account with the same roles.
+
+## Metabase: adding a BigQuery dataset
+
+Once you have your created and downloaded your service account JSON file for your BigQuery dataset, head over to your Metabase instance, click on the **settings** cog, and select **Admin** to bring up Admin mode. In the **Databases** section, click on the **Add database** button in the upper right. (You can also access this page from the **Settings** menu, under **Setup** and **GET CONNECTED**.)
+
+On the **ADD DATABASE** page, select **BigQuery** from the **Database type** dropdown. Metabase will present you with the relevant configuration fields to fill out:
+
+![images](../images/bigquery_add_database.png)
+
+### Fields
+
+#### Name
+
+**Name** is the title of your database in Metabase.
+
+#### Dataset ID
+
+Each BigQuery dataset will have a **Dataset ID**. You can find this ID via the [Google Cloud Console](https://console.cloud.google.com/). If you're not sure where to find the Dataset ID, see Google's documentation on [getting information on datasets](https://cloud.google.com/bigquery/docs/dataset-metadata#getting_dataset_information).
+
+> When entering the Dataset ID, omit the Project ID prefix. For example, if your ID is `project_name:dataset_id`, only enter `dataset_id`.
+
+#### Service account JSON file
+
+Upload the service account JSON file you created when following the [steps above](#google-cloud-platform:-creating-a-service-account-and-json-file). The JSON file contains the credentials your Metabase application will need to read and query your dataset, as defined by the **roles** you added to the service account. If you need to add additional roles, you have to create another service account, download the JSON file, and upload the file to Metabase.
+
+### Sliders
+
+#### Use the Java Virtual Machine (JVM) timezone
 
-### Connecting to a BigQuery Dataset
+_Default: Disabled_
```

Comment:
What's the rationale behind documenting the default state, particularly since we give advice below about leaving it off?

Reply:
This text is provided in the UI itself. I'm including it in the docs as well so that readers will have the information even when they are not following along while configuring a connection (for example, if they are exploring our docs to gauge our support for BigQuery).